src/sidebar/util/time.js

```diff
@@ -151,40 +147,47 @@ const BREAKPOINTS = [
  *
  * @param {Date} date - The date to consider as the timestamp to format.
  * @param {Date} now - The date to consider as the current time.
- * @return {breakpoint} A dict that describes how to format the date.
+ * @return {breakpoint|null} An object that describes how to format the date or
+ *                           null if no breakpoint matches.
  */
 function getBreakpoint(date, now) {
-  let breakpoint;
-  for (let i = 0; i < BREAKPOINTS.length; i++) {
-    breakpoint = BREAKPOINTS[i];
+  for (let breakpoint of BREAKPOINTS) {
```

Comment:
Out of curiosity I had a look at the transpiled output of `for ... of`. Interestingly it uses a short version if the value can be statically determined to be an array, but the full `Symbol.iterator`-using iteration process otherwise. This means that `for ... of` could be somewhat slower in hot loops.